### PR TITLE
fix node4 build issue

### PR DESCRIPTION
**Pull request by ahmadnassri**
fixes: #541

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "description": "Accessibility engine for automated Web UI testing",
   "version": "3.0.0-alpha.4",
   "license": "MPL-2.0",
+  "engines": {
+    "node": ">=4"
```
Comment (marcysutton) [inline package.json]: Do we need to specify a node version if we're downgrading it to use `var`?

**Comment by renatoi** [comment]
Maybe also add supported node version to test to catch these issues in the future:
https://github.com/dequelabs/axe-core/blob/develop/circle.yml

**Comment by marcysutton** [comment]
@renatoi our tests running on Circle have different requirements than axe-core for end-users. We transpile lib JS using Babel for backwards-compatibility but this isn't required for our tests. The postinstall script wasn't caught by Babel unfortunately.

File: package.json (hunk 1 above)
Comment (ahmadnassri) [reply package.json]: it just produces a warning for anybody running on older versions, so it's useful

**Pull request merged**